model/event.go

```diff
@@ -101,6 +101,7 @@ type EventMeta struct {
 	RootID     string `json:"root_id"`
 	Group      string `json:"group"`
 	Created    string `json:"created"`
+	SchemaName string `json:"schema_name"`
```

Comment:
Is there any reason for not just using the entity type? I intended to replace ContentEntity with the actual schema name acknowledging that having ConentEntity in the that field was not very useful